src/Commands/AbstractDrupalCommands.php

```diff
@@ -378,6 +379,19 @@ public function settingsSetup(array $options = [
             $this->getConfig()
         )->setConfigKey('drupal.settings');
 
+        // If ran in dev mode copy local settings file to 'settings.local.php'.
+        // Such file will be conditionally included in 'settings-override-file'.
+        if ($options['dev']) {
+            $examples_settings_path = $this->getConfig()->get('drupal.site.settings_local_file');
```

Comment:
(optional, could be ignored) May be could be renamed to
```suggestion
            $settings_local_file = $this->getConfig()->get('drupal.site.settings_local_file');
```